homeassistant/components/motioneye/config_flow.py

```diff
@@ -42,13 +43,18 @@ def _get_form(
             user_input: dict[str, Any], errors: dict[str, str] | None = None
         ) -> FlowResult:
             """Show the form to the user."""
+            url_schema: dict[vol.Required, type[str]] = {}
```

Comment:
I think it should be more generic rather than this specific case.
```suggestion
            url_schema: dict[vol.Marker, type] = {}
```

Reply:
Why would I make this less specific?

Reply:
In case someone decide to change this schema, this type should work with arbitrary schema. But in this particular case it doesn't really matter.

Reply:
ok 🤷